www/js/init.js

```diff
@@ -25,9 +25,11 @@
 require.config({
     baseUrl: 'js/lib',
     paths: {
-        'jquery': 'jquery-2.1.4',
+        //'jquery': 'jquery-2.1.4', // Implements #268
+        //'jquery': 'jquery-3.2.1', // Full version
+        'jquery': 'jquery-3.2.1.slim', // Slim version
```

Comment:
The packaged version of jQuery comes with a version number, so I think that's why it has version. It's probably useful to future developers to be able to see at a glance which version is being used. I think especially if we use .slim future developers will need to be able to see this at a glance. I would therefore recommend keeping. (It may be the case that, with so many versions of jQuery floating around the Web, the developers of jQuery decided to use explicit version numbering in their releases.)